app.js

```diff
@@ -0,0 +1,67 @@
+/* eslint-disable eqeqeq */
+let books = [];
+const booksList = document.getElementById('bookcard');
+
+window.addEventListener('load', () => {
+  const content = JSON.parse(localStorage.getItem('books'));
+  if (content === 0 || content === null) {
+    localStorage.setItem('books', JSON.stringify(books));
+  } else {
+    books = content;
```

Comment:
[Optional] I think that there was no need to add a listener to the window on load. Any script call directly within the file will be executed on load of the webpage. I would've wrapped the code block within a function and called the function directly within the script.